internal/lua-sandbox/api/api.go

```diff
@@ -0,0 +1,575 @@
+// Copyright The gittuf Authors
+// SPDX-License-Identifier: Apache-2.0
+
+// This file contains modified code from the lua-sandbox project, available at
+// https://github.com/kikito/lua-sandbox/blob/master/sandbox.lua, and licensed
+// under the MIT License
+
+package sandbox
+
+import (
+	"bufio"
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"regexp"
+	"strconv"
+	"strings"
+	"time"
+
+	sandbox "github.com/gittuf/gittuf/internal/lua-sandbox/util"
+	lua "github.com/yuin/gopher-lua"
+)
+
+// Lua sandbox helper functions and wrapper functions of provided Go APIs
+var luaHelpers = `
+function splitString(str, sep)
+	if sep == nil then
+		sep = "\n"
+	end
+
+	local lines = {}
+	for line in string.gmatch(str, "([^"..sep.."]+)") do
+		table.insert(lines, line)
+	end
+
+	return lines
+end
+
+function set_instruction_quota(quota)
+    local count = 0
+
+    debug.sethook(function()
+        count = count + 1
+        if count > quota then
+            error("Instruction quota exceeded (" .. quota .. " instructions).", 2)
+        end
+    end, "", 1)
+end
+
+function goLinter(path) return runFunc("goLinter", path, nil) end
+function removeFile(path) return deleteFile(path) end
```

Comment:
we're effectively going to provide both `removeFile` and `deleteFile` to a user. Let's get rid of that duplication.